getpaid/backends/transferuj/__init__.py

```diff
@@ -23,7 +23,13 @@ class PaymentProcessor(PaymentProcessorBase):
 
     _GATEWAY_URL = u'https://secure.transferuj.pl'
     _REQUEST_SIG_FIELDS = (u'id', u'kwota', u'crc',)
-    _ALLOWED_IP = (u'195.149.229.109', )
+    _ALLOWED_IP = (
+        u'195.149.229.109',
+        u'148.251.96.163',
```

Comment:
Możliwe że 195.149.229.109 trzeba usunąć bo wygląda jakby był używany jako smtp.
Pozostałych revDNS wskazuje na tpay ale jak masz gdzies pod ręką link do docy z tymi ipkami to wklej proszę?